src/components/AwardCard.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <v-card class="d-flex align-center" :color="theme.neutral.lighten2" :data-tournament="`${place}${suffix}`">
+  <v-card class="d-flex align-center" :color="theme.neutral-lighten2" :data-tournament="`${place}${suffix}`">
```

Comment:
This isn't related to these tests but it was referencing a color that didn't exist and bombing the modal.